style.css

```diff
@@ -148,6 +148,434 @@ body {
+  background: #F29F9F;
+}
+
+.form__colors-item:nth-of-type(3):before {
+  background: #A8BFF9;
+}
+
+.form__colors-item:nth-of-type(4):before {
+  background: #90BD8F;
+}
+
+.middle-section__colors-and-price {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: justify;
+  -ms-flex-pack: justify;
+  justify-content: space-between;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+  margin: 0;
+  height: 46px;
+  min-width: 250px;
+  margin-bottom: 22px;
+}
+
+.middle-section__order {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+  padding: 0;
+}
+
+.middle-section__order-item {
+  width: 256px;
+  padding-top: 9px;
+  padding-bottom: 26px;
+  margin-bottom: 10px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+  text-align: center;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+  cursor: pointer;
+}
+
+.middle-section__price {
+  margin: 0;
+  margin-right: 8px;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 28px;
+}
+
+.middle-section__subtitle {
+  text-transform: uppercase;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 10px;
+  line-height: 10px;
+  letter-spacing: 0.07em;
+  color: #000000;
+  opacity: 0.5;
+}
+
+.middle-section__order-item {
+  border: 1.5px solid #000;
+  box-sizing: border-box;;
+}
+
+button {
+  background: none;
+}
+
+.middle-section__btn,
+.middle-section__btn_other {
+  text-transform: uppercase;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 13px;
+}
+
+.middle-section__btn_busket {
+  width: 203px;
+  height: 30px;
+  padding-top: 14px;
+  border: 1.5px solid #000000;
+  text-align: center;
+}
+
+.middle-section__title {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 20px;
+  color: #000000;
+}
+
+.middle-section__desription {
+  font-style: normal;
+  font-weight: normal;
+  font-size: 9px;
+  line-height: 13px;
+  color: #000000;
+  opacity: 0.5;
+}
+
+.bottom-section {
+  position: relative;
+  padding: 33px 12px;
+  background: -webkit-gradient(linear, left top, right top, color-stop(27%, #EBE3D8), color-stop(27%, #000), color-stop(38%, #000), color-stop(38%, #EBE3D8));
+  background: -webkit-linear-gradient(left, #EBE3D8 27%, #000 27%, #000 38%, #EBE3D8 38%);
+  background: -o-linear-gradient(left, #EBE3D8 27%, #000 27%, #000 38%, #EBE3D8 38%);
+  background: linear-gradient(to right, #EBE3D8 27%, #000 27%, #000 38%, #EBE3D8 38%);
+}
+
+.bottom-section__image {
+  z-index: -50;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  justify-content: center;
+}
+
+.bottom-section__img {
+  width: 100%;
+  -webkit-box-shadow: -11px 25px 40px -34px rgba(0,0,0,0.8);
+  box-shadow: -11px 25px 40px -34px rgba(0,0,0,0.8);
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/WebHeroSchool/locoalin/pullRequest?prid=5835770)